packages/cli/src/commands/dns/rm.ts

```diff
@@ -75,7 +84,7 @@ function readConfirmation(
     output.print(
       `${chalk.bold.red('> Are you sure?')} ${chalk.gray('(y/N) ')}`
     );
-    process.stdin
+    client.stdin
```

Comment:
We can't easily expect on `process`, so needed to swap to `client.stdin`. This changed the arguments to this function a tiny bit.